torch/nn/modules/loss.py

```diff
@@ -1131,6 +1131,9 @@ class probabilities only when a single class label per minibatch item is too res
             of smoothing when computing the loss, where 0.0 means no smoothing. The targets
             become a mixture of the original ground truth and a uniform distribution as described in
             `Rethinking the Inception Architecture for Computer Vision <https://arxiv.org/abs/1512.00567>`__. Default: :math:`0.0`.
+        dim (int, optional): Specifies the dimension of the classes (the dimension across which softmax is computed).
+            If :attr:`dim` is set to ``None``, then :attr:`dim` defaults to 1.
```

Comment:
could you update this docstring to mention that it is not used for the no-batch-dim case as well`(C)`